recipes/conda-forge-build-setup/meta.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: conda-forge-build-setup
+  version: 1.0.0
+
+build:
+  number: 0
+  script:
+    - IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\activate.d MKDIR %LIBRARY_PREFIX%\etc\conda\activate.d  # [win]
+    - COPY %RECIPE_DIR%/activate.bat %LIBRARY_PREFIX%\etc\conda\activate.d\conda-forge-build-setup_activate.bat  # [win]
+    - mkdir -p "${PREFIX}/etc/conda/activate.d"  # [unix]
+    - cp "${RECIPE_DIR}/activate.sh" "${PREFIX}/etc/conda/activate.d/conda-forge-build-setup_activate.sh"  # [unix]
+
+requirements:
+  run:
+    - conda-build
+    - jinja2
+    - anaconda-client
+
+test: {}
+
+about:
+  home: https://github.com/conda-forge/conda-forge-build-setup-feedstock
+  summary: A package installed by conda-forge each time a build is run on CI. This package has side-effects to your conda config.
+  license: BSD 3-clause
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - ocefpaf
```

Comment:
Ping @ocefpaf. Let me know if you'd rather not be on this one, but I suspect you won't mind.